### PR TITLE
show logging CLI support for logs stored in tmpfs

### PR DESCRIPTION
**Pull request by mihirpat1**
Signed-off-by: Mihir Patel <patelmi@microsoft.com>

#### What I did
Enabling "show logging" CLI to show logs from tmpfs if it exists.

#### How I did it
Added a check in the CLI handler to see if the path "/var/log.tmpfs" exists and display the logs from this directory if it exists. In case if the path doesn't exist, logs will be displayed from "/var/log"

#### How to verify it
Following tests were performed:
1.	"show logging" o/p when syslog resides in "/var/log"
2.	"show logging -f" o/p when syslog resides in "/var/log"
3.	"show logging" o/p when syslog resides in "/var/log.tmpfs"
4.	"show logging -f" o/p when syslog resides in "/var/log.tmpfs"

For detailed test results, please refer to [Unit-test_logging_cli.txt](https://github.com/sonic-net/sonic-utilities/files/10550147/Unit-test_logging_cli.txt)

### CODE REVIEW

**Review by prgeor — APPROVED** [review]

**Comment by prgeor** [comment]
@gechiang can you review?

File: tests/show_test.py [hunk 1]
```diff
@@ -49,3 +52,50 @@ def teardown_class(cls):
...
+        ]
+)
+def test_show_logging_syslog_1(run_command, cli_arguments, expected):
+    runner = CliRunner()
+    result = runner.invoke(show.cli.commands["logging"], cli_arguments)
+    run_command.assert_called_with(EXPECTED_BASE_COMMAND + expected, display_cmd=False)
+
+@patch('show.main.run_command')
+@patch('os.path.exists', MagicMock(return_value=True))
+@pytest.mark.parametrize(
```
Comment (gechiang) [inline tests/show_test.py:89]: Can you also add a test case to handle syslog.1 exists under the log.tmpfs directory as well?

Comment (mihirpat1) [reply tests/show_test.py:89]: @gechiang, I have added the test case now.

**Review by gechiang — APPROVED** [review]
LGTM
Thaks for fixing this!

**Comment by yxieca** [comment]
Rejecting this change for 201811 branch.

**Comment by qiluo-msft** [comment]
This commit could not be cleanly cherry-picked to 202012. Please submit another PR.

**Pull request merged**